sdk/eventhub/Azure.Messaging.EventHubs/stress/stress-test-resources.bicep

```diff
@@ -10,25 +10,49 @@ param testApplicationOid string
 @description('The location of the resource group.')
 param location string = resourceGroup().location
 
-var eventHubsNamespace_var = resourceGroup().name
+var namespaceName = resourceGroup().name
+var shortNamespaceName = substring(replace(namespaceName, '-',''), 0, 15)
```

Comment:
You might also consider using the `guid()` function here to avoid possible name overlaps when shortening. Though it may be simpler to wait for us to add unique short name support into addons instead.